File: BE/Src/MIT.Fwk.Infrastructure/Entities/UserRole.cs
using JsonApiDotNetCore.Resources;
using JsonApiDotNetCore.Resources.Annotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MIT.Fwk.Infrastructure.Entities
{
    [Resource]
    [Table("AspNetUserRoles")]
    public class UserRole : Identifiable<string>
    {


        [Attr]
        public string RoleId { get; set; }
        [Attr]
        public string UserId { get; set; }

        [Attr]
        public int TenantId { get; set; }

        [HasOne]
        public virtual User User { get; set; }

        [HasOne]
        public virtual Role Role { get; set; }

        [HasOne]
        public virtual Tenant Tenant { get; set; }
    }
}

File: FE/WEB/app/models/user-role.js
import Model, { attr, belongsTo } from '@ember-data/model';

export default class UserRoleModel extends Model {
  @belongsTo('user', { async: true, inverse: 'userRoles' }) user;
  @belongsTo('role', { async: true, inverse: 'userRoles' }) role;

  @attr('string', { defaultValue: '' }) roleId;
  @attr('string', { defaultValue: '' }) userId;
  @attr('number', { defaultValue: 0 }) tenantId;
}
